src/components/generics/DocumentRating.vue

```diff
@@ -163,6 +163,11 @@ export default {
         },
       };
     },
+	formatBoulderRating(rating) {
+		if (this.document.climbing_outdoor_type !== 'bloc') return rating;
```

Comment:
I would say that it is not so good to have this hardcoded `'bloc'` here, would be better to have a centralized variable imported in components, but it would be quite a refactoring I believe ^^'